cmd/acra-keymaker/acra-keymaker.go

```diff
@@ -67,12 +67,26 @@ func main() {
 	cmd.ValidateClientID(*clientID)
 
 	if *masterKey != "" {
-		newKey, err := keystore.GenerateSymmetricKey()
+		var newKey []byte
+		switch *keystoreVersion {
+		case "v1":
+			newKey, err = keystore.GenerateSymmetricKey()
+		case "v2":
+			newKey, err = keystoreV2.NewSerializedMasterKeys()
+		case "":
+			log.Errorf("Key store version is required: --keystore={v1|v2}")
```

Comment:
```suggestion
			log.Errorf("Key store version is required: --keystore={v1|v2}. Latest is v2.")
```

Reply:
I'm pretty sure the users can tell that v1 < v2 😄

If we really want to suggest a choice to the users, I'd add another log line with info level, describing the preference. Or provide a link to documentation (sadly, we don't have it right now) so that they can make an informed choice without being constrained to 50 characters of a log message.

I'm also not sure if I'd recommend switching to v2 right away. How about postponing this decision until the release?

Reply:
Info log + postponing decision makes sense :)